scripts/install_third_party_libs.py

```diff
@@ -26,29 +26,25 @@
 TOOLS_DIR = os.path.join(os.pardir, 'oppia_tools')
 
 
-def install_prerequisites():
-    """Install libraries that are required before running or
-    importing any script.
-    """
-    tools_dir = os.path.join(os.pardir, 'oppia_tools')
-    prerequisites = [
-        ('pyyaml', '5.1.2', os.path.join(tools_dir, 'pyyaml-5.1.2')),
-        ('future', '0.17.1', os.path.join('third_party', 'future-0.17.1')),
-    ]
-
-    for package, version, path in prerequisites:
-        if not os.path.exists(path):
-            command = [
-                sys.executable, '-m', 'pip', 'install', '%s==%s'
-                % (package, version), '--target', path]
-            uextention = ['--user', '--prefix=', '--system']
-            process = subprocess.Popen(
-                command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-            if 'can\'t combine user with prefix' in process.communicate()[1]:
-                subprocess.check_call(command + uextention)
-
+"""Install libraries that are required before running or
+importing any script.
+"""
+PREREQUISITES = [
+    ('pyyaml', '5.1.2', os.path.join(TOOLS_DIR, 'pyyaml-5.1.2')),
+    ('future', '0.17.1', os.path.join('third_party', 'future-0.17.1')),
+]
+
+for PACKAGE, VERSION, PATH in PREREQUISITES:
+    if not os.path.exists(PATH):
+        COMMAND = [
+            sys.executable, '-m', 'pip', 'install', '%s==%s'
+            % (PACKAGE, VERSION), '--target', PATH]
+        UEXTENTION = ['--user', '--prefix=', '--system']
+        PROCESS = subprocess.Popen(
```

Comment:
ditto

Reply:
I think variables outside a function have to be in this pattern and I couldnt find any workaround other than disabling it so I had to make it capitals..

Reply:
you missed this variable :)

Reply:
Fixed!